api/lib/utils.js

```diff
@@ -48,6 +49,11 @@ function genGameId() {
     return Math.floor(Math.random() * max) + 10000;
 }
 
+function genTeamId() {
+    let max = 900000;
+    return Math.floor(Math.random() * max) + 100000;
```

Comment:
Just make a helper function that you pass the max to cuz It's a duplicate of the one above with different numbers

Reply:
yeah sure